src/MX/MegaMenu/Controller/Adminhtml/Menu/Export.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MX\MegaMenu\Controller\Adminhtml\Menu;
+
+use MX\MegaMenu\Controller\Adminhtml\Menu as MenuController;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Framework\Registry;
+
+class Export extends MenuController
+{
+    /**
+     * @var PageFactory
+     */
+    protected $resultPageFactory;
+
+    /**
+     * Constructor.
+     *
+     * @param Context $context
+     * @param Registry $registry
+     * @param PageFactory $resultPageFactory
+     */
+    public function __construct(
+        Context $context,
+        Registry $registry,
+        PageFactory $resultPageFactory
+    )
+    {
+        parent::__construct($context, $registry);
+        $this->resultPageFactory = $resultPageFactory;
+    }
+
+    public function execute()
+    {
+        $resultPage = $this->resultPageFactory->create();
```

Comment:
you can use $this->resultFactory->create(ResultFactory::TYPE_PAGE); in which case you can remove the constructor